### DISCUSSION TITLE
error 500 on log in

### DISCUSSION THREAD

**Discussion by jenny787** [mediacms-io/mediacms#738] [Q&A]
single server install on Ubuntu
been running for over two years without update
the site is working as in you can play the videos, but you can't log in.

I get an error 500 when I try to log in.

**Comment by mgogoulos** [comment]
Can you provide some sort of logs? Without it it's difficult to tell what can be happening. 
Can you get access to the server, do a `tail -f logs*` , try to login at that time, and get the log messages that should contain the error?

**Comment by awilfox** [comment]
I had this happen when Redis wasn't running.  Ensuring Redis was running fixed that error.

**Comment by Dos7t3r** [comment]
same problem…… how to solve it 🧐

**Reply by acrawford73** [reply]
```bash
sudo systemctl status redis
```

Should say 'active'

```bash
redis-server.service - Advanced key-value store
     Loaded: loaded (/lib/systemd/system/redis-server.service; enabled; vendor preset: enabled)
     Active: active (running)
```

If not 'active', then:

```bash
sudo systemctl restart redis
```

**Reply by Dos7t3r** [reply]
> ```bash
> 
> sudo systemctl status redis
> 
> ```
> 
> 
> 
> Should say 'active'
> 
> 
> 
> ```bash
> 
> redis-server.service - Advanced key-value store
> 
>      Loaded: loaded (/lib/systemd/system/redis-server.service; enabled; vendor preset: enabled)
> 
>      Active: active (running)
> 
> ```
> 
> 
> 
> If not 'active', then:
> 
> 
> 
> ```bash
> 
> sudo systemctl restart redis
> 
> ```

Thanks for helping
this is how i solved it：
https://stackoverflow.com/questions/9736975/django-admin-doesnotexist-at-admin
